test/functional/fixtures/api/es-next/custom-client-scripts/testcafe-fixtures/i4122.js

```diff
@@ -0,0 +1,10 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `Fixture`
+    .clientScripts({ content: 'window["test"] = 0;' });
+
+const getTestValue = ClientFunction(() => window['test']);
+
+test('test', async t => {
+    await t.expect(getTestValue()).eql(1);
```

Comment:
This test doesn't check that both scripts are injected, does it?

Reply:
Yes, it is. I can rewrite this.

Reply:
let's rewrite it little bit